pageobjects/helpWidget.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+module.exports = {
+  url: function useEnvironmentUrl() {
+    return this.api.launch_url;
+  },
+  /* Help widget commands located here. The help widget iframe elements cannot be mapped with the
+  current page objects api from nightwatch. */
+  commands: [{
+    accessHelpWidget() {
+      return this.api
+        .frame('launcher')
+        .click('button[aria-label="Help"]')
+        .frame(null)
+        .frame('webWidget')
+        .waitForElementVisible('input[type="search"]', 5000, 'help widget expanded');
+    },
+    helpWidgetSearch(searchTerm, browser) {
+      return this.api
+        .setValue('input[type="search"]', searchTerm)
+        .keys(browser.Keys.ENTER)
+        .assert.visible('a[href^="https://support.tidepool.org"]', 'support articles showing');
+    },
+    helpWidgetContactUsBtn() {
+      return this.api
+        .useXpath()
+        .click('//button[text()="Contact us"]')
+        .useCss()
+        .waitForElementVisible('input[name="name"]', 5000, '"contact us" form displayed');
+    },
+    helpWidgetContactUsFormFill() {
+      return this.api
+        .setValue('input[name="name"]', 'Automated UI Testing')
+        .setValue('input[name="email"]', 'webuiautomation@tidepool.org')
```

Comment:
should we pull this email from config?

Reply:
great suggestion . I was having a hard time with it because its in an iframe so I had thrown in the white flag. I decided to try it one more time this morning and I was able to get it! thanks for pointing this out! I was also able to simplify some of the methods and rename them with something less redundant then "helpWidget" (we are in the helpwidget page object, probably don't need to keep referring to it as such)